Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/RolesController.cs

```diff
@@ -246,11 +246,11 @@ public HttpResponseMessage GetRoleUsers(string keyword, int roleId, int pageInde
                         Localization.GetString("InvalidRequest", Components.Constants.LocalResourcesFile));
                 }
 
-                var users = RoleController.Instance.GetUserRoles(this.PortalId, Null.NullString, role.RoleName);
+                var users = RoleController.Instance.GetUserRoles(this.PortalId, Null.NullString, role.RoleName).OrderBy(x => x.FullName).ToList();

```

Comment:
Can we move the `OrderBy` call to the beginning of line 260? This would allows us to only specify it once, and avoid creating another copy of the list.

Reply:
yes of course, I fix it.